src/features/pass-survey/model/hooks/useActivityStepper.ts

```diff
@@ -34,14 +36,23 @@ function useActivityStepper(state: ActivityState | undefined) {
     answers[step]?.additionalAnswer != null &&
     answers[step]?.additionalAnswer !== '';
 
+  const invalidTimeRangeAnswer =
+    isTimeRangeStep &&
+    ((answers[step]?.answer as TimeRangeResponse)?.startTime === null ||
+      (answers[step]?.answer as TimeRangeResponse)?.endTime === null);
```

Comment:
The check is correct. However, I'd like to propose some refactoring. Here is my idea:
We have an `AnswerValidator` instance here. We could add a new method called `isValidAnswer` that could perform the check-ups described above for the Time Range item and just check for undefined | null for other items.
Then, we could have something like this:
`const canMoveNext = canContinue && answerValidator.isValidAnswer();`